resources/postinstall.js

```diff
@@ -0,0 +1,31 @@
+var osenv = require('osenv');
+var fs = require('fs-extra');
+var keypair = require('keypair');
+
+var dir = osenv.home() + '/.tessel';
+var filename = 'id_rsa';
```

Comment:
You might consider changing the name of this file so it's not the default name for an SSH key. It's a very minor point but it could be confusing if users are, for some reason, looking at two keys with the same name in different folders.